packages/ra-ui-materialui/src/layout/UserMenu.js

```diff
@@ -36,23 +38,24 @@ class UserMenu extends React.Component {
     };
 
     render() {
-        const { children, label, logout, translate } = this.props;
+        const { children, label, icon, logout, translate } = this.props;
         if (!logout && !children) return null;
         const { anchorEl } = this.state;
         const open = Boolean(anchorEl);
 
+        const menuIconProps = {
```

Comment:
I don't need you need to extract the props now that only the `icon` prop is accepted.